submit-web/src/components/NewManagementPlan/PlanDetails.tsx

```diff
@@ -30,15 +31,32 @@ export const PlanDetails = ({ onSubmit }: PlanDetailsProps) => {
 
   return (
     <TabBox title="Create New Submission">
-      <When condition={Boolean(existingPlan) && !newlyCreatedPlan}>
-        <ExistingPlanDetails existingPlan={existingPlan} />
-      </When>
-      <When condition={!existingPlan}>
-        <NewPlanDetails
-          onSubmit={onSubmit}
-          setNewlyCreatedPlan={setNewlyCreatedPlan}
-        />
-      </When>
+      <If condition={isLoading}>
+        <Then>
+          <Grid
+            container
+            sx={{
+              padding: "16px 0px",
+            }}
+            spacing={3}
+          >
+            <Grid item xs={12}>
```

Comment:
early returns are easier to read.
![image](https://github.com/user-attachments/assets/89485d38-8551-44c5-9deb-3ff25876cc74)